src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts

```diff
@@ -0,0 +1,13 @@
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+import { WidgetCategory } from 'app/pages/dashboard/types/widget-category.enum';
+import { dashboardWidget } from 'app/pages/dashboard/types/widget-component.interface';
+import { SlotSize } from 'app/pages/dashboard/types/widget.interface';
+import { WidgetCpuModelComponent } from 'app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.component';
+
+export const cpuModelWidget = dashboardWidget({
+  name: T('CPU Model'),
+  component: WidgetCpuModelComponent,
+  category: WidgetCategory.SystemInfo,
```

Comment:
Should be under CPU category.

Reply:
it was there initially, then I thought it's better to move it to System 😄

Reply:
✅ 
<img width="403" alt="Screenshot 2024-08-15 at 18 15 09" src="https://github.com/user-attachments/assets/830d395f-f317-4063-9afa-c36cd46ff02e">